service/schemas/recipe.js

```diff
@@ -1,8 +1,68 @@
 // npm modules
 import { Schema, model } from "mongoose";
 
-const recipe = new Schema();
+const ingredientSchema = new Schema({
+    measure: {
+        type: String,
+        required: true
+    }
+});
 
-const Recipe = model("ingredient", recipe);
+const recipeSchema = new Schema({
+    title: { 
+        type: String,
+        required: true
+    },
+    category: {
+        type: String,
+        enum: [ "Beef", "Breakfast", "Chicken", "Dessert", "Goat", "Lamb", "Miscellaneous", "Pasta", "Pork", "Seafood", "Side", "Starter", "Vegan", "Vegetarian"],
+        required: true
+    },
+    area: {
+        type: String,
+        required: true
+    },
+    instructions: {
+        type: String,
+        required: true
+    },
+    description: {
+        type: String
+    },
+    thumb: {
+        type: String,
+        validate: /^(http|https):\/\/[^\s]+$/
+    },
+    preview: {
+        type: String,
+        validate: /^(http|https):\/\/[^\s]+$/
+    },
+    time: {
+        type: Number,
+        required: true
+    },
+    favorites: [{ 
+        type: mongoose.Types.ObjectId,
+        ref: 'User'
+    }],
+    youtube: {
+        type: String,
+        validate: /^(http|https):\/\/[^\s]+$/
+    },
+    tags: [{ 
+        type: String 
+    }],
+    createdAt: {
```

Comment:
do this like that:  { versionKey: false, timestamps: true },

Reply:
Ok